language/bytecode_verifier/src/type_memory_safety.rs

```diff
@@ -829,13 +829,15 @@ impl<'a> TypeAndMemorySafetyAnalysis<'a> {
                 }
             }
 
-            Bytecode::BorrowGlobal(idx, type_actuals_idx) => {
+            // TODO: Handle type actuals for generics
```

Comment:
Could you delete this comment? This now supports generics.

Reply:
> LGTM. One thing I'm unsure. @runtian-zhou is it still required to copy paste the script hashes?

Yea I think so. Though I'm not sure if this diff will change script hashes tho as the simple transaction scripts won't use those bytecode primitives.